packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 46.9.0 - August 8, 2019
+
+## Added
+
+#### salesforcedx-utils-vscode
+
+- Use VS Code api to open browser in host machine when operating in container mode ([PR #1448](https://github.com/forcedotcom/salesforcedx-vscode/pull/1448))
+
+#### docs
+
+- Add Japanese translations for docs ([PR #1528](https://github.com/forcedotcom/salesforcedx-vscode/pull/1528))
```

Comment:
@praksb, Do we have a location where the Japanese version of the docs will be uploaded? In which case, could we include that link here?
Include Japanese translation for Salesforce Extensions for VS Code documentation <add link>

Reply:
I think this is the link. let me add it.
https://github.com/forcedotcom/salesforcedx-vscode/tree/develop/docs/articles-jp